corehq/apps/app_manager/templates/app_manager/v1/partials/case_config_advanced.html

```diff
@@ -178,7 +178,7 @@ <h3 class="panel-title" data-bind="html: header"></h3>
                 <div class="form-group">
                     <label>{% trans "Case Type" %}</label>
                     <select class="form-control" data-bind="
-                        options: config.caseTypes,
+                        options: caseConfig.caseTypes,
```

Comment:
Do https://github.com/dimagi/commcare-hq/blob/eb893f0091554b485f8cb26aa205fe6d1e25f2e5/corehq/apps/app_manager/templates/app_manager/v1/partials/case_config_advanced.html#L30 and https://github.com/dimagi/commcare-hq/blob/eb893f0091554b485f8cb26aa205fe6d1e25f2e5/corehq/apps/app_manager/templates/app_manager/v1/partials/case_config_advanced.html#L155 also need to use `caseConfig`?